Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -321,8 +321,11 @@ static ASCIILiteral serviceName(const ProcessLauncher::LaunchOptions& launchOpti
     // 1. When the application and system frameworks simply have different localized resources available, we should match the application.
     // 1.1. An important case is WebKitTestRunner, where we should use English localizations for all system frameworks.
     // 2. When AppleLanguages is passed as command line argument for UI process, or set in its preferences, we should respect it in child processes.
+#if PLATFORM(MACCATALYST)
     auto initializationMessage = adoptOSObject(xpc_dictionary_create(nullptr, nullptr, 0));
     _CFBundleSetupXPCBootstrap(initializationMessage.get());
+    xpc_connection_send_bootsrap(m_xpcConnection.get(), initializationMessage.get());
```

Comment:
bootsrap?

Reply:
This should be `xpc_connection_set_bootstrap`, I believe.